lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua

```diff
@@ -0,0 +1,20 @@
+-- PvP Functions
+E2Helper.Descriptions["playerIsInPvp"]     = "Returns 1 if a player is in Pvp, and 0 otherwise."
+E2Helper.Descriptions["playerIsInBuild"]   = "Returns 1 if a player is in build, and 0 otherwise."
+E2Helper.Descriptions["getPvpStatus"]      = "Returns \"pvp\" if a player is in Pvp, and \"build\" if they are in build."
+E2Helper.Descriptions["getPvpers"]         = "Returns a table of all players currently in Pvp."
+E2Helper.Descriptions["getBuilders"]       = "Returns a table of all players currently in Build."
+
+-- Ping Functions
+E2Helper.Descriptions["averagePing"] = "Returns a floating-point number representing the average ping of players in the server."
+
+-- String Functions
+E2Helper.Descriptions["startsWith"] = "Returns a 1 if the string on which it was called starts with start, and 0 otherwise."
+
+-- Array Functions
+E2Helper.Descriptions["shuffled"]   = "Returns a randomly shuffled version of an array."
+E2Helper.Descriptions["difference"] = "Returns an array filled with the differing elements between the calling and argument arrays."
+E2Helper.Descriptions["reverse"]    = "Returns a reversed array."
```

Comment:
We have `shuffled` and `reverse`. We should make them both verbs or both adjectives to stay consistent.